runtimes/mlflow/mlserver_mlflow/runtime.py

```diff
@@ -151,7 +151,10 @@ async def invocations(
 
     async def load(self) -> bool:
         # TODO: Log info message
-        model_uri = await get_model_uri(self._settings)
+        model_uri = await get_model_uri(
+            self._settings,
+            allowed_schemes=mlflow.store.artifact.artifact_repository_registry._artifact_repository_registry._registry,
+        )
         self._model = mlflow.pyfunc.load_model(model_uri)
```

Comment:
I don't have much experience downloading things from `dbfs://` / `s3://` using MLflow, but would `load_model()` just handle everything else? How would we configure the DBFS / S3 host, credentials, etc?

Reply:
In the case of S3 URLs, MLflow retrieves the artifact through `boto3`, which retrieves the credentials from an environment variables `AWS_ACCESS_KEY_ID` or the default credential store `~/.aws`.